src/lib/tasks/server/execute_server_prepared_statement_task.cpp

```diff
@@ -10,7 +11,7 @@ namespace opossum {
 void ExecuteServerPreparedStatementTask::_on_execute() {
   try {
     const auto tasks = OperatorTask::make_tasks_from_operator(_prepared_plan, CleanupTemporaries::Yes);
-    CurrentScheduler::schedule_and_wait_for_tasks(tasks);
+    Hyrise::get().current_scheduler.schedule_and_wait_for_tasks(tasks);
```

Comment:
Now that we access it through `Hyrise`, does "current" bring anything to the table?